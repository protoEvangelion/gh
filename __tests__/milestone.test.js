/**
 * Â© 2013 Liferay, Inc. <https://liferay.com> and Node GH contributors
 * (see file: CONTRIBUTORS)
 * SPDX-License-Identifier: BSD-3-Clause
 */

'use strict'

const { runCmd } = require('./testUtils')

describe('E2E: Milestone Module Test', () => {
    it('List Milestones `gh ms --list`', done => {
        expect(runCmd('bin/gh.js ms --list')).toMatchSnapshot()
        done()
    })
})
