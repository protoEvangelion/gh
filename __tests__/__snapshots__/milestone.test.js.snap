// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E: Milestone Module Test List Milestones \`gh ms --list\` 1`] = `
"Listing milestones on protoEvangelion/gh
 Milestone 2 milestone 2 description @open (due 5 months ago)
 Milestone 1 milestone 1 description @open (due 5 months ago)
"
`;
